src/iplant_groups/routes/schemas/group.clj

```diff
@@ -8,6 +8,7 @@
 (s/defschema GroupStub (group-schema/group-stub "group"))
 (s/defschema GroupWithDetail (group-schema/group-with-detail "group"))
 (s/defschema GroupList (group-schema/group-list "group" "groups"))
+(s/defschema GroupListWithDetail (group-schema/group-list-with-detail "group" "groups"))
```

Comment:
It doesn't look like `GroupList` is used anywhere else, so this new `GroupListWithDetail` could probably just become `GroupList` now.

Reply:
On the other hand, we already have `Group` vs. `GroupWithDetail`, so I can see keeping both for consistency.